tests/integration/pilot/gateway_conformance_test.go

```diff
@@ -93,17 +102,32 @@ func TestGatewayConformance(t *testing.T) {
 					Insert(suite.MeshCoreFeatures.UnsortedList()...)
```

Comment:
I never got the chance to add a stable channel set upstream, so mind adding the GRPCRoute tests here?

Reply:
done